src/locale/pages/organizeJourneyInstance/en.yml

```diff
@@ -1,3 +1,7 @@
+addAssigneeButton: Add assignee
+addMemberButton: Add member
+addMemberLabel: Add member
```

Comment:
I think the keys should be `addSubjectButton` and `addSubjectLabel`. The English localizations can potentially remain "member" if we can't come up with a more neutral term (maybe "person"?).

We have traditionally avoided using the term "member" in the interface and API, because that word means different things in different organizations. Not every person in Zetkin is a member of the organization where they're organized.

Reply:
Person sounds good! I thought about this also, because "assignee" is such an obvious word, but for member there is no real equvialent - I agree person is good!